payment.yaml

```diff
@@ -0,0 +1,227 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Payment Ecosystem
+  description: Internal API that provides payment and wallet creation services
+servers:
+  - url: 'https://api.kinmarketplace.internal/v1/'
+paths:
+  /orders/{order_id}:
+    parameters:
+      - $ref: '#/components/parameters/RequestId'
+      - in: path
+        name: order_id
+        schema:
+          type: string
+        required: true
+        description: application order id
+    get:
+      tags:
+        - Orders
+      description: get order info
+      summary: get order info
+      operationId: getOrder
+      responses:
+        '200':
+          description: Successfully returned order info
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Order'
+        '404':
+          description: Order not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        default:
+          description: Generic Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /orders:
+    parameters:
+      - $ref: '#/components/parameters/RequestId'
+    post:
+      tags:
+        - Orders
+      description: create an payment order
+      summary: create an payment order
+      operationId: createOrder
+      requestBody:
+        description: payment description
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Payment'
```

Comment:
should be the same schema as in POST, here it's Payment, there it's Order

Reply:
There is no way in swagger to give the same object with different requirements.  So I prefer creating two different objects for the request and response